deploy/openshift/quay-app.yaml

```diff
@@ -427,4 +427,4 @@ parameters:
     displayName: aws load balancer connection idle timeout
     description: aws load balancer connection idle timeout
   - name: DB_CONNECTION_POOLING
-    value: false
+    value: "false"
```

Comment:
Switch this back to "true" while we're at it

Reply:
done!